velox/dwio/parquet/tests/duckdb_reader/ParquetTpchTest.cpp

```diff
@@ -169,43 +174,98 @@ std::unordered_map<std::string, std::string> ParquetTpchTest::duckDbParquetWrite
         R"(COPY (SELECT * FROM {}) TO '{}' (FORMAT 'parquet', ROW_GROUP_SIZE {}))")};
 
 TEST_F(ParquetTpchTest, Q1) {
-  assertQuery(1, 2, 10);
+  queryId_ = 1;
```

Comment:
Oh, sorry. Let's not include these changes in this PR. Let's make a new one. BTW, using member variables doesn't help much. It is actually redundant to use member variables and pass these to assertQuery. Using number for the query ID is fine. It is clear that q1 passes 1 for the query id. The problem is with expectedPipelineCount_ and expectedFinishedSplits_. I wonder if these are needed at all.

Reply:
Got it, thanks for the clarification. Yes, I thought they were redundant too, will check if the expectedPipelineCount_ and expectedFinishedSplits_ validations can be simplified. Reverting these changes.